simulation/ether.js

```diff
@@ -0,0 +1,3 @@
+export default function ether (n) {

```

Comment:
Do we really need this function?

Reply:
nope, don't understand how it added to project. i'll delete it